packages/host-portal-app/src/types.ts

```diff
@@ -9,6 +9,7 @@ export type AppRegView = {
 export interface Manifest {
   module: string;
   style?: string;
```

Comment:
As this demo is heavily tied to the micro frontend interface from Modular, should this type be exported by Modular-scripts?

Reply:
I think it's a good idea. I also have another branch to factor out the library, let's chat tomorrow to coordinate!